inventories/e2e-inventory.yaml

```diff
@@ -0,0 +1,33 @@
+vars:
```

Comment:
2 questions: 

Why is the agent inventory in `inventory.yaml` and these two images in the `inventories` dir?

If we keep these here, can we remove `-inventory` from the filename? They are already in the `inventories` dir, we already know what they are